zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -559,21 +581,23 @@ def channel_message_to_zerver_message(realm_id: int, users: List[ZerverFieldsT],
                                       all_messages: List[ZerverFieldsT],
                                       zerver_realmemoji: List[ZerverFieldsT],
                                       zerver_subscription: List[ZerverFieldsT],
-                                      added_channels: AddedChannelsT,
+                                      added_channels: AddedChannelsT, id_list: List[int],
```

Comment:
Would this be better as a `Tuple[int, int, int, int]`?

Reply:
`Tuple[int, int, int, int]` gives an error: `541: error: Incompatible types in assignment (expression has type "Tuple[int, int, int, int]", variable has type "List[int]")`

Reply:
See my comment above -- you actually need to make it a tuple (e.g. `(a, b, c)`, not `[a, b, c]`) as well.